src/Polly.Core/CircuitBreaker/BreakDurationGeneratorArguments.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Polly.CircuitBreaker;
+
+/// <summary>
+/// Represents arguments used to generate a dynamic break duration for a circuit breaker.
+/// </summary>
+public class BreakDurationGeneratorArguments
+{
+    public BreakDurationGeneratorArguments(
```

Comment:
Add `context` parameter at the begining and also `Context` property of type `ResilienceContext`.